rustfmt-config/src/options.rs

```diff
@@ -235,6 +244,17 @@ impl WidthHeuristics {
             single_line_if_else_max_width: (50.0 * max_width_ratio).round() as usize,
         }
     }
+
+    pub fn maximum(max_width: usize) -> WidthHeuristics {
+        WidthHeuristics {
+            fn_call_width: max_width,
+            struct_lit_width: max_width,
+            struct_variant_width: max_width,
+            array_width: max_width,
+            chain_width: max_width,
+            single_line_if_else_max_width: max_width,
```

Comment:
I would use `usize::max_value()` rather than `max_width` means one less arg, plus no ordering constraint on initialisation.

Reply:
fwiw, the `scaled()` constructor also takes max_width, but I can still change this np.

Reply:
`scaled` needs it though since it uses the max width as input (we could probably use `max_value` there for the 100% ones).